sklearn/cluster/_bicluster.py

```diff
@@ -30,8 +30,8 @@ def _scale_normalize(X):
     X = make_nonnegative(X)
     row_diag = np.asarray(1.0 / np.sqrt(X.sum(axis=1))).squeeze()
     col_diag = np.asarray(1.0 / np.sqrt(X.sum(axis=0))).squeeze()
```

Comment:
I want to check a bit more to know if we could raise a more informative warning instead of generic zero division one.